src/aiogram_dialog/widgets/kbd/pager.py

```diff
@@ -42,6 +42,8 @@ class PagerPageData(PagerData):
 DEFAULT_PAGE_TEXT = Format("{target_page1}")
 DEFAULT_CURRENT_PAGE_TEXT = Format("[ {current_page1} ]")
 
+DEFAULT_PAGER_ROW_LENGTH = 8
```

Comment:
let's add it as parameter for `NumberedPager.__init__` with default to old logic